packages/Breadcrumbs/src/components/Link/Link.js

```diff
@@ -19,26 +20,53 @@ function Link(props) {
   const isDark = React.useContext(IsDarkContext);
   const I18n = useI18n();
   const shouldTruncate = isString(children) && children.length > MAXIMUM_NUM_OF_CHARACTER;
-
-  const link = (
-    <sc.Link data-pka-anchor="breadcrumbs.link" as={as} kind="minor" href={href} isDark={isDark} {...moreProps}>
-      {hasOnlyOneChild ? <sc.ArrowIcon aria-label={I18n.t("breadcrumbs.aria_back_to")} /> : null}
-      {shouldTruncate ? truncate(children) : children}
-    </sc.Link>
-  );
+  const isUsingDefaultLinkComponent = !as;
+  const commonComponentProps = {
+    "data-pka-anchor": "breadcrumbs.link",
+    as: isUsingDefaultLinkComponent ? Button.Link : as,
+    href: href || undefined,
+    isDark,
+  };
+  const defaultLinkComponentProps = isUsingDefaultLinkComponent
+    ? {
+        ...commonComponentProps,
+        kind: Button.types.kind.MINOR,
+      }
+    : commonComponentProps;
```

Comment:
IMO this would be more readable:

```js
const extraProps = {};

if (isUsingDefaultLinkComponent) {
  extraProps['kind'] = Button.types.kind.MINOR;
}

<sc.Link
    data-pka-anchor="breadcrumbs.link"
    as={isUsingDefaultLinkComponent ? Button.Link : as}
    href={href || undefined}
    isDark
    {...extraProps}
```